pkg/controller/migmigration/annotation.go

```diff
@@ -184,7 +206,14 @@ func (t *Task) labelNamespaces(client k8sclient.Client) error {
 			"NS annotations/labels added.",
 			"name",
 			namespace.Name)
+
+		if time.Now().Sub(now) == time.Second*3 {
```

Comment:
@JaydipGabani to validate how this change works with large migrations, I'd suggest using [kube-kong-operator](https://github.com/konveyor/kube-kong-operator) to deploy a few thousand resources into your namespaces. If you have questions about using kube-kong, @pranavgaikwad should be able to help you get started.

Once we do this, we can see how many reconciles it takes to get through annotation of all resources.